tests/nodes/NodeConnectionManager.termination.test.ts

```diff
@@ -262,35 +252,35 @@ describe(`${NodeConnectionManager.name} termination test`, () => {
       ).rejects.toThrow(nodesErrors.ErrorNodeConnectionTimeout);
     } finally {
       await nodeConnectionManager?.stop();
-      await revProxy?.stop();
+      await proxy?.stop();
       await server?.serverClose();
     }
   });
   test('closed based on bad certificate during withConnection', async () => {
     let server;
     let nodeConnectionManager: NodeConnectionManager | undefined;
-    let revProxy: ReverseProxy | undefined;
+    let proxy: Proxy | undefined;
     try {
       server = tcpServer();
-      revProxy = new ReverseProxy({
+      proxy = new Proxy({
         logger: logger,
+        authToken: '',
       });
       await server.serverListen(0);
-      await revProxy.start({
+      await proxy.start({
         serverHost: server.serverHost(),
         serverPort: server.serverPort(),
-        ingressHost: '127.0.0.1' as Host,
+
```

Comment:
We need to set an `ingressHost`. its `proxyHost` now. Check the rest of the file for this.